projects/fsastorefrontlib/src/cms-components/my-account/inbox/inbox-tab/inbox-messages/inbox-messages.component.ts

```diff
@@ -161,7 +164,9 @@ export class InboxMessagesComponent implements OnInit, OnDestroy {
       });
     if (selectedMessages.length > 0) {
       this.subscription.add(
-        this.inboxService.setMessagesState(selectedMessages, toRead).subscribe()
+        this.inboxService
+          .setMessagesState(selectedMessages, toRead)
+          .subscribe(data => this.inboxService.setUnreadMessageState(true))
```

Comment:
Use pipe -> tap, and subscribe at the end. No need to do logic inside subscribe.

Reply:
I already fix to pipe, just need to change map -> tap.